recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ requirements:
     - expat
     - libgcc
     - esmf
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20|0.2.20.*
```

Comment:
Even though `0.2.20` is available we are not pinning to that version b/c we could not build `scipy` with it.
To ensure ecosystem stability we should revert this to `openblas 0.2.19|0.2.19.*`.

PS: the pinning info is located [here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39-L86).

Reply:
Thank you for pointing me to that script.  I now see how the pinned versions are chosen.